lib/rmt/mirror.rb

```diff
@@ -4,6 +4,29 @@
 require 'time'
 
 class RMT::Mirror
+  class FileReference
+    class << self
+      def build_from_metadata(metadata, base_dir:)
+        new(base_dir: base_dir, relative_remote_path: metadata.location)
+          .tap do |file|
+            file.arch = metadata.arch
+            file.checksum = metadata.checksum
+            file.checksum_type = metadata.checksum_type
+            file.size = metadata.size
+          end
+      end
+    end
+
+    attr_reader :local_path, :relative_remote_path
+    attr_accessor :arch, :checksum, :checksum_type, :size
+    alias location relative_remote_path
```

Comment:
You aren't using `relative_remote_path` anywhere outside the class, drop the alias use `location` everywhere instead.

Reply:
Done :heavy_check_mark: